src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -328,6 +341,16 @@ public virtual void WriteLine(char[] buffer, int index, int count)
             WriteLine();
         }
 
+        public virtual void WriteLine(ReadOnlySpan<char> source)
+        {
+            char[] buffer = ArrayPool<char>.Shared.Rent(source.Length);
+
+            source.CopyTo(new Span<char>(buffer, 0, source.Length));
```

Comment:
nit, you can just do `source.CopyTo(new Span<char>(buffer));`

Reply:
done...